src/components/food-item-block/FoodItemBlock.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import "../food-item-block/FoodItemBlock.css";
+import whooperImage from "../food-item-block/whooper.png";
+
+function FoodItemBlock() {
```

Comment:
same as FoodBlock send food item as props
and destructure keys

```suggestion
function FoodItemBlock({foodItem}) {
const {name, calories, weight, description, price} = foodItem
}
```